inc/saiswitch.h

```diff
@@ -2429,12 +2431,24 @@ typedef enum _sai_switch_attr_t
 
     /**
      * @brief Packet action when a packet ingress and gets routed back to same tunnel
+     * This enum is only available in v1.7.
      *
      * @type sai_packet_action_t
      * @flags CREATE_AND_SET
      * @default SAI_PACKET_ACTION_FORWARD
      */
-    SAI_SWITCH_ATTR_TUNNEL_LOOPBACK_PACKET_ACTION,
+    SAI_SWITCH_ATTR_TUNNEL_LOOPBACK_PACKET_ACTION = 186,
```

Comment:
this is the attribute that is only available in v1.7 but not in master, then if there is any other attribute with the same value that might be added in master, how can we handle that?